### PR TITLE
rake task generates fake games, games transfer points on save

### PR DESCRIPTION
**Pull request by vpontis**

### CODE REVIEW

File: app/models/game.rb [hunk 1]
```diff
@@ -39,15 +40,15 @@ def loser_usernames=(usernames)
...
     if @winner_usernames
-      winner_not_found = @winner_usernames - winner_usernames
+      total_not_found += @winner_usernames - winner_usernames
     end
     if @loser_usernames
-      loser_not_found = @loser_usernames - loser_usernames
+      total_not_found += @loser_usernames - loser_usernames
     end
 
-    total_not_found = winner_not_found + loser_not_found
```
Comment (stubailo) [inline app/models/game.rb:49]: nice!

File: app/views/players/index.html.haml [hunk 2]
```diff
@@ -11,5 +11,5 @@
       %td= player.ranking
       %td
         %strong= link_to player.username, player
-      %td= player.points
+      %td= number_with_precision(player.points, precision: 2)
```
Comment (stubailo) [inline app/views/players/index.html.haml]: change this to show integers only

File: lib/tasks/populate.rake [hunk 3]
```diff
@@ -1,28 +1,33 @@
...
+    p.password_confirmation = 'passpass'
+    p.skip_confirmation!
+    p.save!
+
+
+    25.times do |n|    
       p = Player.new
-      p.username = "Player#{n}"
-      p.email = "player_#{n}@gmail.com"
+      p.username = Faker::Name.name
```
Comment (stubailo) [inline lib/tasks/populate.rake:16]: This is fine I guess but it's going to be hard to fill in game forms fast for testing

Comment (vpontis) [inline lib/tasks/populate.rake:16]: No it's actually pretty easy in practice.

File: lib/tasks/populate.rake [hunk 4]
```diff
@@ -1,28 +1,33 @@
...
-    # users = User.all(limit: 6)
-    # 50.times do
-    #   body = Faker::Lorem.sentence(5)
-    #   users.each { |user| user.posts.create!(body: body) }
-    # end
+    50.times do |n|
+      g = Game.new
+      players = Player.order("RANDOM()")[0,4]
+      g.winners = players[2,2]
+      g.losers = players[0,2]
```
Comment (stubailo) [inline lib/tasks/populate.rake:28]: holy shit I've never seen this syntax before

**Comment by stubailo** [comment]
Just make it automatically merge by merging in develop, and change it to show integers for points and it'll be good

**Pull request merged**